Emby.Server.Implementations/Library/Resolvers/BaseVideoResolver.cs

```diff
@@ -156,19 +156,27 @@ protected void SetIsoType(Video video)
                 }
                 else
                 {
-                    // use disc-utils, both DVDs and BDs use UDF filesystem
-                    using (var videoFileStream = File.Open(video.Path, FileMode.Open, FileAccess.Read))
-                    using (UdfReader udfReader = new UdfReader(videoFileStream))
+                    try
                     {
-                        if (udfReader.DirectoryExists("VIDEO_TS"))
+                        // use disc-utils, both DVDs and BDs use UDF filesystem
+                        using (var videoFileStream = File.Open(video.Path, FileMode.Open, FileAccess.Read))
+                        using (UdfReader udfReader = new UdfReader(videoFileStream))
                         {
-                            video.IsoType = IsoType.Dvd;
-                        }
-                        else if (udfReader.DirectoryExists("BDMV"))
-                        {
-                            video.IsoType = IsoType.BluRay;
+                            if (udfReader.DirectoryExists("VIDEO_TS"))
+                            {
+                                video.IsoType = IsoType.Dvd;
+                            }
+                            else if (udfReader.DirectoryExists("BDMV"))
+                            {
+                                video.IsoType = IsoType.BluRay;
+                            }
                         }
                     }
+                    catch (Exception)
+                    {
+                        // something really bad happened
```

Comment:
Failing silently is never good. Can we log here?

Reply:
There's no `_logger` in that class, nor in the `MovieResolver` which inherits it. We could print into stderr, but then it would be hidden too. I can add the _logger attribute in the `MovieResolver` and pass it to the `BaseVideoResolver` for logging.

Reply:
Updated with using `Video.Logger`